test/tools/ossfuzz/StackReuseCodegenFuzzer.cpp

```diff
@@ -75,8 +75,7 @@ DEFINE_PROTO_FUZZER(Program const& _input)
 	YulStringRepository::reset();
 
 	solidity::frontend::OptimiserSettings settings = solidity::frontend::OptimiserSettings::full();
-	settings.runYulOptimiser = false;
-	settings.optimizeStackAllocation = false;
+	settings.copyYulSettingsFrom(solidity::frontend::OptimiserSettings::minimal());
```

Comment:
@bshastry Is this supposed to be literally "optimizer disabled" or just the default, minimal optimization? I'm assuming the latter but let me know if you think I should change it.

Reply:
No longer relevant. Now Yul optimizer cannot be disabled even via `OptimiserSettings`, so I have no choice but to do it like this.